tests/Unit/DataStructures/Tensor/EagerMath/Test_CrossProduct.cpp

```diff
@@ -0,0 +1,186 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cmath>
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/CrossProduct.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+SPECTRE_TEST_CASE("Unit.DataStructures.Tensor.EagerMath.EuclideanCrossProduct",
```

Comment:
also show trivial things like x cross y = z

Reply:
I have added x cross y = z to my other test. I don't know how to do the python testing framework, because I don't know how to recover the valence of the input (is it a vector or a one form), which the function needs to know.